pkg/utils/common.go

```diff
@@ -179,27 +178,27 @@ func querySQL(db *sql.DB, query string, maxRetry int) (*sql.Rows, error) {
 
 	for i := 0; i < maxRetry; i++ {
 		if i > 0 {
-			log.Warnf("sql query retry %d: %s", i, query)
+			log.L().Warn("query retry", zap.Int("retry number", i), zap.String("query", query))
 			time.Sleep(retryTimeout)
+		} else {
+			log.L().Debug("query sql", zap.String("query", query))
 		}
 
-		log.Debugf("[query][sql]%s", query)
-
 		rows, err = db.Query(query)
 		if err != nil {
-			log.Warnf("[query][sql]%s[error]%v", query, err)
+			log.L().Warn("query retry", zap.String("query", query), log.ShortError(err))
 			continue
 		}
 
 		return rows, nil
 	}
 
 	if err != nil {
-		log.Errorf("query sql[%s] failed %v", query, errors.ErrorStack(err))
+		log.L().Error("query retry", zap.String("query", query), log.ShortError(err))
```

Comment:
here is query failed at last, no more retry.
btw the logic is redundant, the `err` is always not nil in L196